Emby.AutoOrganize/Core/EpisodeFileOrganizer.cs

```diff
@@ -434,29 +427,39 @@ private NamingOptions GetNamingOptionsInternal()
             FileOrganizationResult result,
             CancellationToken cancellationToken)
         {
-            var episode = await GetMatchingEpisode(series, seasonNumber, episodeNumber, endingEpiosdeNumber, result, premiereDate, cancellationToken);
+            var episode = await GetMatchingEpisode(
+                series,
+                seasonNumber,
+                episodeNumber,
+                endingEpiosdeNumber,
+                result,
+                premiereDate,
+                cancellationToken
+            ).ConfigureAwait(false);
```

Comment:
This should be at the end of the last parameter.

Reply:
Addressed in 77ff4a3cf7c9be911d8154059c55fea9b010d07c